Lib/ufo2ft/filters/dottedCircleFilter.py

```diff
@@ -0,0 +1,158 @@
+import logging
+import math
+
+from ufoLib2.objects import Glyph
+
+from ufo2ft.featureCompiler import parseLayoutFeatures
+from ufo2ft.featureWriters import ast
+from ufo2ft.filters import BaseFilter
+from ufo2ft.util import _GlyphSet, _LazyFontName
+
+logger = logging.getLogger(__name__)
+CIRCULAR_SUPERNESS = 0.551784777779014
+
+
+def average(lst):
```

Comment:
You could probably use https://docs.python.org/3/library/statistics.html?highlight=mean#statistics.mean here.